### DISCUSSION TITLE
Using schema registry with client serializers and deserializers in nodejs

### DISCUSSION THREAD

**Discussion by ivar891** [Apicurio/apicurio-registry#2612] [Q&A]
we are using nodejs/nestjs for our app and want to use apicurio registry for storing schemas and encoding and decoding (serializing and deserializing) messages ,but I dont find any sdks or rest apis for serializing and deserializing in nodejs , is it supported , if it is not what are the available alternatives for javascript/nodejs ecosystem

**Comment by carlesarnal** [comment]
I'm sorry @ivar891 as of today, the only implementation of the serdes we have is for java. There are no alternatives available in javascript. We do have rest clients in other languages (javascript, python and go) but those only allow you to interact with registry on base operations (create, delete, list artifacts etc), no serialization features are available there.
